src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -98,6 +103,9 @@ public function __construct()
 
         // Add the default, local resource search path:
         $this->searchPaths[] = __DIR__ . "/../Resources";
+
+        // root path for ordinary composer projects
+        $this->applicationRootPath = dirname(dirname(dirname(dirname(dirname(dirname(__DIR__))))));
```

Comment:
Whoever uses `TemplateHelper::shorten` wants this default to work, so let’s move this line into the constructor of `TemplateHelper`.